src/seccomp.c

```diff
@@ -81,17 +84,59 @@ int pg_init_seccomp(void)
 		ALLOW_SYSCALL(gettimeofday),
 		ALLOW_SYSCALL(stat),
 		ALLOW_SYSCALL(clock_gettime),
+		ALLOW_SYSCALL(mprotect),
+		ALLOW_SYSCALL(rt_sigreturn),
+		ALLOW_SYSCALL(epoll_create),
+		ALLOW_SYSCALL(epoll_ctl),
+		ALLOW_SYSCALL(epoll_wait),
+		ALLOW_SYSCALL(getsockopt),
+		ALLOW_SYSCALL(setsockopt),
+		ALLOW_SYSCALL(readlink),
+		ALLOW_SYSCALL(prlimit64),
+		ALLOW_SYSCALL(memfd_create),
+		ALLOW_SYSCALL(timerfd_create),
+		ALLOW_SYSCALL(uname),
+		ALLOW_SYSCALL(iopl),
 
-		KILL_PROCESS,
```

Comment:
I think this should be configurable, so take a bitmask in function parameter, and use that parameter here

Reply:
it is already defined in seccomp-bpf 
# define SECCOMP_RET_TRAP	0x00030000U /* disallow and force a SIGSYS */

Reply:
sure, but filter contain ```TRAP_PROCESS```
Which doesn't work, if you want packetgraph to KILL process.
You can't put ```KILL_PROCESS``` into filter either, because that would mean packetgraph doesn't work if you want it to print process.
So you need  to have a way to chose at runtime between KILL or TRAP.